cohorts/2023/leaderboard.md

```diff
@@ -628,5 +628,19 @@ Links:
 <td><a href="https://github.com/ChungWasawat/dtc_de_project">Project</a></td>
 <td> <a href="https://www.linkedin.com/in/wasawat-boonyarittikit-b1698b179/"><img src="https://user-images.githubusercontent.com/875246/192300614-2ce22ed5-bbc4-4684-8098-d8128d71aac5.png" height="16em" /></a> <a href="https://github.com/ChungWasawat"><img src="https://user-images.githubusercontent.com/875246/192300611-a606521b-cb76-4090-be8e-7cc21752b996.png" height="16em" /></a></td>
 <td></td>
+</tr>
+<tr>
+<td>Fedor Faizov</td>
+<td><a href="https://github.com/Fedrpi/de-zoomcamp-bandcamp-project">Project</a></td>
+<td> <a href="https://www.linkedin.com/in/fedor-faizov-a75b32245/"><img src="https://user-images.githubusercontent.com/875246/192300614-2ce22ed5-bbc4-4684-8098-d8128d71aac5.png" height="16em" /></a> <a href="https://github.com/Fedrpi"><img src="https://user-images.githubusercontent.com/875246/192300611-a606521b-cb76-4090-be8e-7cc21752b996.png" height="16em" /></a></td>
+<td>
+<details>
+<summary>More info</summary>
+
+
+
+> Absolutly amazing course <3 </details></td>
+</td>
```

Comment:
I think this one is not needed. Can you please double check?

Reply:
yes, you're right. last </td> is not needed. fixed it